printing/app.yaml

```diff
@@ -0,0 +1,10 @@
+port: 50052
+
+application: beachshirts
+host: source3
+cluster: us-west
+service: printing
+shard: primary
+
+tags:
+  location: Palo-Alto
```

Comment:
might want to add another custom tag 
say
 env: Staging or env: Production 

do it elsewhere too